pkgs/os-specific/linux/libratbag/default.nix

```diff
@@ -12,14 +12,14 @@ stdenv.mkDerivation rec {
     rev    = "v${version}";
     sha256 = "sha256-TQ8DVj4yqq3IA0oGnLDz+QNTyNRmGqspEjkPeBmXNew=";
   };
-
```

Comment:
nit: I wouldn't remove this line, for readability

Reply:
Not sure where it went honestly